content/download.html

```diff
@@ -147,6 +150,7 @@ <h3>Community-maintained packages</h3>
       <pre>$ xbps-install openra</pre>
     </dd>
   </dl>
+  <p class="downloadthirdparty">We do not maintain this external package source, so there may be delays when a new version is released.<br />Please contact the downstream repository maintainers about any packaging issues.</p>
```

Comment:
This is duplicating the disclaimer that is already stated above!

Reply:
I removed the statement above.